### ISSUE TITLE
Add colors to test outputs to make them more readable

### ISSUE THREAD

**Issue by eladmallel** [livepeer/go-livepeer#587]
**Is your feature request related to a problem? Please describe.**
My problem is that I find the current test outputs hard to parse.

**Describe the solution you'd like**
I'd love to see test results in colors, similar to the examples in [this link](https://stackoverflow.com/questions/27242652/colorizing-golang-test-run-output).

Of the ideas in the above link I liked [richgo](https://github.com/kyoh86/richgo) the most because the setup looks simple enough, and it's actively maintained.

**Describe alternatives you've considered**
Another approach that could work is the [first solution](https://stackoverflow.com/a/27245610) to the same SO question above, which involves piping test outputs to sed. This solution could also work, but I prefer richgo because it colors more stuff by default.

**Additional context**
Here's what richgo looks like on my machine running `richgo test -v`:

![image](https://user-images.githubusercontent.com/1137044/47687257-d3932700-dbb5-11e8-940b-e71a29a8567f.png)

**Comment by samparsky** [comment]
@eladmallel is this still needed? I can work on it

**Comment by eladmallel** [comment]
@samparsky IMHO it's a nice-to-have, but you're welcome to go for it!
